src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -21,7 +22,7 @@ internal enum Stamp
             SetToMaxUsed = 3,
         }
 
-        private readonly List<Cookie> m_list = new List<Cookie>();
+        private readonly ArrayList m_list = new ArrayList();
```

Comment:
Yes. Currently all the types which already had correct variable names (which we didn't rename from netfx) are not commented yet. I would do this after the crucial part is done and the deadline is met...